contracts/FlowIDTableStaking.cdc

```diff
@@ -834,10 +834,24 @@ pub contract FlowIDTableStaking {
             self.delegatorRewards[delegatorID] = rewards
         }
     }
+
+    /// Interface that only contains operations that are part
+    /// of the regular automated functioning of the epoch process
+    /// These are accessed by the `FlowEpoch` contract through a capability
+    pub resource interface EpochOperations {
+        pub fun setEpochTokenPayout(_ newPayout: UFix64)
+        pub fun setSlotLimits(slotLimits: {UInt8: UInt16})
+        pub fun setNodeWeight(nodeID: String, weight: UInt64)
```

Comment:
```suggestion
```
These two are not used by `FlowEpoch`, from what I can tell. I'm wondering if they should be part of this interface.

Reply:
They're not used right now, but I could see them needing to be used in the future when we have permissionless node operation and slashing. Do you think we should just wait until then to include them?

Reply:
I'm actually happy to include them, you're right that they'll likely be used eventually. 👍